test/tables.txt

```diff
@@ -1202,7 +1285,341 @@ Here's a link to [Freedom Planet 2][].
 <tr>
 <td align="center">r3c1</td>
 <td align="center">r3c2</td>
-</tr></tbody></table>
+</tr>
+</tbody>
+</table>
+````````````````````````````````
+
+### Tables inside of blockquotes
```

Comment:
These aren't part of the ticket, but the new parsing logic makes this behave the same as GitHub, so I added a bunch of tests to make sure it works like GitHub